src/_includes/layouts/token-documentation.njk

```diff
@@ -1,24 +1,30 @@
----
-layout: "layouts/base.njk"
----
+{% extends "layouts/base.njk" %}
```

Comment:
"Extend" allows us to overwrite block content. I use it here to 
- render tabs and header before content
- Set the page title to the following pattern: tab - token - base